fe-vite-react/src/App.jsx

```diff
@@ -1,34 +1,40 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState} from 'react'
+import {Route, Routes, Link} from "react-router-dom";
+import NewsList from './NewsList';
+import SingleArticlePage from './SingleArticlePage';
+
+
+
+const Header = () => {
+  return (
+    <div className='header'>
+  <h1 className='title'>Latest NC News </h1>
+  </div>
+  )
+};
 
 function App() {
-  const [count, setCount] = useState(0)
 
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+
+      <>
+        <Header /> 
+          <nav>
+          <Link to ="/">Home</Link> | <Link to ="/articles">All News</Link> |
+            <Link to ="/articles/topics/football"> Football </Link> |
+             <Link to ="/articles/topics/coding">Coding </Link> 
+           | <Link to ="/articles/topics/cooking">Cooking</Link> 
+        </nav>
```

Comment:
this can easily be a Nav component extracted away instead of being written in-line

Reply:
Done